docs: update cloud data sources README links (#37820)
grafana/grafana: public/app/plugins/datasource/cloudwatch/README.md

> # CloudWatch Data Source - Native Plugin

Reviewer: ```suggestion
# CloudWatch data source - native plugin
```

> Grafana ships with **built in** support for CloudWatch. You just have to add it as a data source and you will be ready to build dashboards for your CloudWatch metrics.

Reviewer: ```suggestion
Grafana ships with **built in** support for CloudWatch. Simply add it as a data source and are ready to build dashboards for your CloudWatch metrics.
```
